storage/buckets/main_test.go

```diff
@@ -17,6 +17,7 @@ import (
 var (
 	storageClient *storage.Client
 	bucketName    string
+	kmsKeyName    string
```

Comment:
We'll talk about testing later, but `kmsKeyName` is only used in one location. So, it's OK to define it there.